examples/react/getting-started/src/App.tsx

```diff
@@ -40,17 +49,24 @@ export function App() {
           searchClient={searchClient}
           indexName="instant_search"
           insights={true}
+          disposeMode={disposeMode}
```

Comment:
we probably need to set up an updater for this parameter, and in vue too (after it's accepted)

Reply:
I was thinking instead of handling prop update, we could warn (like we do with `searchClient`), we probably don't want users to mess with this at runtime, wdyt?

Reply:
yes, however we could also just update the flag likely (until we have an InstantSearch options update method that is also used in constructor 👍)